pkg/compute/store/types.go

```diff
@@ -91,4 +91,7 @@ type ExecutionStore interface {
 	UpdateExecutionState(ctx context.Context, request UpdateExecutionStateRequest) error
 	// DeleteExecution deletes an execution
 	DeleteExecution(ctx context.Context, id string) error
+	// GetExecutionCount returns a count of all executions that completed
+	// successfully on this compute node
+	GetExecutionCount(ctx context.Context) int
```

Comment:
You could argue that this should return a `uint` to more accurately describe the count – it shouldn't go below zero, right?